### PR TITLE
Misc. bag of code cleanups

### PR DESCRIPTION
**Pull request by ValarDragon**

### CODE REVIEW

File: x/pool-incentives/keeper/distr.go [hunk 1]
```diff
@@ -92,28 +92,14 @@ func (k Keeper) SetDistrInfo(ctx sdk.Context, distrInfo types.DistrInfo) {
...
-
-// This is checked for no err when a proposal is made, and executed when a proposal passes
-func (k Keeper) ReplaceDistrRecords(ctx sdk.Context, records ...types.DistrRecord) error {
-	distrInfo := k.GetDistrInfo(ctx)
-
+// Validates a list of records to ensure that:
+// 1) there are no duplicates,
+// 2) the records are in sorted order.
+// 3) the records only pay to gauges that exist
+func (k Keeper) validateRecords(ctx sdk.Context, records ...types.DistrRecord) error {
```
Comment (ValarDragon) [inline x/pool-incentives/keeper/distr.go:99]: This code de-duplication is probably the only interesting part of this PR

**Comment by mattverse** [comment]
Should be good to merge! Most of the comments are out of curiosity :)

File: x/claim/module.go [hunk 2]
```diff
@@ -45,17 +45,11 @@ func (AppModuleBasic) Name() string {
...
-}
-
 func (AppModuleBasic) RegisterLegacyAminoCodec(cdc *codec.LegacyAmino) {
-	types.RegisterCodec(cdc)
 }
 
 // RegisterInterfaces registers the module's interface types
 func (a AppModuleBasic) RegisterInterfaces(reg cdctypes.InterfaceRegistry) {
-	types.RegisterInterfaces(reg)
 }
```
[the remark this answers is not in the document]
Comment (ValarDragon) [reply x/claim/module.go:54]: The old `types.RegisterInterface` was a no-op

[the remark this answers is not in the document]
Comment (mattverse) [reply x/claim/module.go:54]: Ahh i see i see

**Pull request merged**